tests/Sdk/Unit/Trace/ExporterFactoryTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Unit\Trace;
+
+use OpenTelemetry\Contrib as Path;
+use OpenTelemetry\Sdk\Trace\ExporterFactory as ExporterFactory;
+use PHPUnit\Framework\TestCase;
+
+class ExporterFactoryTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function testIfExporterHasCorrectEndpoint()
```

Comment:
Be sure to add a test for invalid input as well. Perhaps even malicious input if you can construct one.

Reply:
When writing you tests, make sure to use inputs that are malicious like certain html entities. There are examples out there that give examples of malicious attacks (especially with sql injections and other such attacks) although I don't know how relevant they will be to an exporter used for tracing.